framework/source/class/qx/bom/webfonts/Validator.js

```diff
@@ -123,6 +128,14 @@ qx.Class.define("qx.bom.webfonts.Validator", {
       apply : "_applyFontFamily"
     },
 
+    /**
+     * Comparison string used to check whether the font has loaded or not.
+     */
+    comparisonString :
+    {
+      nullable : true,
+      init : null
```

Comment:
Agreed, except that if someone is looking at the code and sees the property definition, they won't know what its default is. Not a big deal, but putting it right in the property could make the code easier to read. (When I was reading it, I had to go find usages in order to figure out that there was a static value defined elsewhere.)